### PR TITLE
Frontend of Merchant Verification

### PR DESCRIPTION
**Pull request by vahinim**
Add Verification application that verifies Merchant with 2 basic questions.

### CODE REVIEW

File: frontend/verification/lib/VerificationPage2.dart [hunk 1]
```diff
@@ -0,0 +1,249 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:verification/menuItems.dart';
+import 'package:verification/verificationSuccess.dart';
+import 'package:verification/verificationFailed.dart';
+
+//class verificationPage2 extends StatelessWidget {
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: Please remove commented code.

File: frontend/verification/lib/VerificationPage2.dart [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+//        textTheme: GoogleFonts.montserratTextTheme(
+//          Theme.of(context).textTheme,
+//        ),
+//      ),
+//      home: VerificationHomePage2(),
+//    );
+//  }
+//}
+
+class VerificationPage2 extends StatefulWidget {
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: Add class level comment as to what it consists and does.
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-a-library-level-doc-comment

File: frontend/verification/lib/VerificationPage2.dart [hunk 3]
```diff
@@ -0,0 +1,249 @@
+import 'package:flutter/material.dart';
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: Name this directory according to

https://dart.dev/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores

Give more descriptive names, like business_registration_details_view, business_address_view, etc. to this file and others

File: frontend/verification/lib/VerificationPage2.dart (hunk 2 above)
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: Please give more descriptive name eg. VerificationHomeView, VerificaitonSuccessView.

File: frontend/verification/lib/VerificationPage2.dart [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+  bool isStorePresent = false;
+  bool isBusinessPresent = false;
+
+  void choiceAction(String choice){
+    if(choice == MenuItems.Cancel) {
+      print('Discard Verification');
+      //When ever user clicks on discard verification it opens up verification failed
+      //TODO: Need to return to home page after cancelling verification
+      _showCancelVerificationDialog();
+
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: remove extra newline.
try formatting using
https://dart.dev/guides/language/effective-dart/style#do-format-your-code-using-dartfmt

File: frontend/verification/lib/VerificationPage2.dart [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+          'Verify',
+          style: TextStyle(
+              fontSize: 16.0,
+          ),
+        ),
+      ), // This trailing comma makes auto-formatting nicer for build methods.
+    );
+  }
+
+  void _verifyMerchant(){
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis

File: frontend/verification/lib/VerificationPage2.dart [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+      Navigator.push(context,
+      MaterialPageRoute(builder: (context) => verificationSuccess()));
+    }
+    else{
+      Navigator.push(context,
+      MaterialPageRoute(builder: (context) => verificationFailed()));
+    }
+  }
+
+  Future<void> _showVerifyDialog() async {
```
Comment (trblessy) [inline frontend/verification/lib/VerificationPage2.dart]: https://dart.dev/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis

**Review by trblessy — COMMENTED** [review]
By convention, the description of the PR is a complete sentence, written as though it were an order
'Add dart files for registration views of the Gpay merchant app'

followed by more description of the exact changes

File: frontend/verification/lib/VerificationPage2.dart (hunk 1 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: removed

File: frontend/verification/lib/VerificationPage2.dart (hunk 3 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: changed .dart file names

File: frontend/verification/lib/VerificationPage2.dart (hunk 2 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

File: frontend/verification/lib/VerificationPage2.dart (hunk 6 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

File: frontend/verification/lib/VerificationPage2.dart (hunk 5 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

File: frontend/verification/lib/VerificationPage2.dart (hunk 2 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

File: frontend/verification/lib/VerificationPage2.dart (hunk 4 above)
Comment (vahinim) [reply frontend/verification/lib/VerificationPage2.dart]: done

**Comment by vahinim** [comment]
Please review again. I have completed the above mentioned changes.

**Review by krati-tiwari — APPROVED** [review]

**Comment by vahinim** [comment]
I have removed Merge conflicts. Please review again and then I will merge.

**Pull request merged**